### ISSUE TITLE
Html / xml parser can't handle tags with names longer than 10 chars

### ISSUE THREAD

**Issue by chfritz** [cursorless-dev/vscode-parse-tree#49]
Due to https://github.com/tree-sitter/tree-sitter-html/issues/30

I'm using this extension inside of my sane-indentation extension and after upgrading vscode today to 1.71.1 I now get:
```
[Extension Host] LanguageStillLoadingError: Language 'javascript' is still loading; please wait and try again
	at getTreeForUri (/home/cfritz/.vscode/extensions/pokey.parse-tree-0.21.2/out/extension.js:192:27)
	at getNodeAtLocation (/home/cfritz/.vscode/extensions/pokey.parse-tree-0.21.2/out/extension.js:206:24)
```
Any idea where this is coming from and how to fix it? Do you need to publish a new version or did something break in your recent commits?

**Comment by chfritz** [comment]
I think I've narrowed it down: the errors only occur when I have an HTML file open among my editors!

**Comment by pokey** [comment]
Interesting! Maybe worth trying the tree-sitter cli with the tree-sitter-html parser on the html file to see if it hangs?

**Comment by chfritz** [comment]
I haven't tried that yet, but I've narrowed it down further: it only happens when the said html file is already open when the window (re-)loads *and* the HTML contains [web components, aka. custom tags](https://developer.mozilla.org/en-US/docs/Web/Web_Components).

**Comment by pokey** [comment]
Ah interesting!  Do you have an example?  I tried an html file with web components, and although it didn't parse correctly, it didn't seem to affect other documents in the workspace

**Comment by chfritz** [comment]
Here is the file. And as a reminder: after you've opened it, you need to do a "Developer: Reload Window" to trigger the bug.

```html
<html>
<head>

</head>

<body>
  <script src="http://portal.homedesk.local:8000/running/@transitive-robotics/webrtc-video/dist/webrtc-video-device.js?userId=cfritz&deviceId=f5b1b62bd4"></script>
  <webrtc-video-device
  id="cfritz"
  host="homedesk.local:8000"
  ssl="false"
  text="/dev/video0"
  timeout="1800"
  source="/dev/video0"
  type="v4l2src"
  width="320"
  height="240"
  framerate="15/1"
  bitrate="20"
  />
</body>
</html>
```

**Comment by RedCMD** [comment]
Cause by `<webrtc-video-device ... />`

For some reason it cannot handle tags that are over 10 chars long
`<abcdefghijk>` will break it, but `<abcdefghij>` wont

**Comment by pokey** [comment]
Fwiw here is the issue to track the >10 chars thing https://github.com/tree-sitter/tree-sitter-html/issues/30

**Comment by pokey** [comment]
fixed by 6317e9198e38bdb478f44f16f5d2cc4ff3a80ad8
